openshift/README.md

```diff
@@ -213,13 +214,42 @@ oc rollout -n $NAMESPACE latest dc/<buildname>-master
 
 *Note: Remember to swap out the bracketed values with the appropriate values!*
 
-### Sidecar Logging
+## Sidecar Logging
 
 Our deployment on OpenShift uses a Fluent-bit sidecar to collect logs from the CHEFS application. The sidecar deployment is included in the main app.dc.yaml file.
 Our NodeJS apps output logs to a configurable file path (for example app/app.log ). This is done using using a logger script. For example see our [CHEFS app logger](https://github.com/bcgov/common-hosted-form-service/blob/master/app/src/components/log.js)
 
+The Fluent-bit configuration is kept in the openshift config map [fleunt-bit.cm.yaml](https://github.com/bcgov/common-hosted-form-service/blob/master/openshift/fluent-bit.cm.yaml)
```

Comment:
"fluent" instead of "fleunt". Also can do a relative link of `openshift/fluent-bit.cm.yaml` instead of the full github URL iirc.